### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/WFC3/uvis_time_dependent_photometry/uvis_timedep_phot.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#330]
## What
An accessibility scan flagged 3 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/WFC3/uvis_time_dependent_photometry/uvis_timedep_phot.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<table border="1" class="dataframe">` (selector: `#calculate-countrates > .cell.container.docutils:nth-child(6) > .cell_output.container.docutils > .text_html.output > div > table`)
- `<table border="1" class="dataframe">` (selector: `.cell.container.docutils:nth-child(4) > .cell_output.container.docutils > .text_html.output > div > table`)
- `<table border="1" class="dataframe">` (selector: `#recompute-countrates > .cell.container.docutils:nth-child(6) > .cell_output.container.docutils > .text_html.output > div > table`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
